GraphicsView/include/CGAL/Qt/qglviewer_impl.h

```diff
@@ -2336,6 +2355,8 @@ CGAL_INLINE_FUNCTION
 void CGAL::QGLViewer::keyReleaseEvent(QKeyEvent *e) {
   if (isValidShortcutKey(e->key()))
     currentlyPressedKey_ = ::Qt::Key(0);
+  if(e->key() == ::Qt::Key_Z && !e->isAutoRepeat())
+    _first_tick = true;
```

Comment:
Why do you need to set `_first_tick` in both `keyPressEvent()` and `keyReleaseEvent()`?